src/services/AsyncStorage.ts

```diff
@@ -129,6 +130,16 @@ const getOverwriteCMS = async (): Promise<CMS | null> => {
   return cms === productionCMS || cms === testCMS ? cms : null
 }
 
+const toggleDevMode = async (): Promise<void> => {
+  const isDevMode = await AsyncStorage.getItem(MODE_KEY)
+  await AsyncStorage.setItem(MODE_KEY, isDevMode ? JSON.stringify(!JSON.parse(isDevMode)) : 'true')
+}
+
+export const getDevMode = async (): Promise<boolean> => {
+  const isDevMode = await AsyncStorage.getItem(MODE_KEY)
+  return isDevMode ? JSON.parse(isDevMode) : false
+}
```

Comment:
I'm a bit conflicted on this. Yes, not set should be null, but the devmode should also be disabled by default (e.g. getDevMode should return false).

I'm also not sure if this doesn't complicate things in the components where it's used.

Reply:
You are right, perhaps its a little bit more complicated. However, just checking for `if (devMode)` should still work since `null` is also falsy. It would also be more consistent with the other async methods as those also return `null` if it was not explicitly set yet. But lets wait for @SteffiStuffels opinion :)

Reply:
I am fine with both.

Reply:
Then your choice @charludo.

Reply:
Alright, `null` doesn't appear to produce any errors or failing test, so I changed it.